### DISCUSSION TITLE
Command constant throttle %

### DISCUSSION THREAD

**Discussion by anoopk18** [bitcraze/discussions#631] [Q&A]
Hi,

I am working on a couple of experiments which require a constant throttle % to be sent to each motor of the single crazyflie that I am testing. Are there easier ways to accomplish this rather than making changes to  firmware.py file itself? I looked through the motors.c file and I was a bit confused by the code provided. With an xbox controller it looks like minor finger movements might alter the thrust commanded, so I was thinking of an 'on' 'off' kind of setup to command this thrust. I realize that it should be making adjustments to the param conds as well. Kindly let me know your thoughts.

**Comment by krichardsson** [comment]
You can use the [`motorPowerSet` parameter group](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/params/#motorpowerset) to control the motors directly
